Framework/PythonInterface/mantid/utils/absorptioncorrutils.py

```diff
@@ -91,6 +91,10 @@ def __get_cache_name(
     cache_filenames = []
     if cache_dirs:
         # generate the property string for hashing
+        try:
```

Comment:
One of these should probably be
```python
height_val_tmp = ws.run().getTimeAveragedValue("BL11A:CS:ITEMS:HeightInContainer")
```
[`Run.getTimeAveragedValue` doc](https://docs.mantidproject.org/nightly/api/python/mantid/api/Run.html#mantid.api.Run.getTimeAveragedValue)

Reply:
Thanks a lot for the comment! Just got it fixed.

Reply:
time-average-value of a scalar is the scalar

Reply:
Oh yes, but do we need any more changes?